sklearn/metrics/tests/test_common.py

```diff
@@ -638,7 +638,8 @@ def test_inf_nan_input():
     for metric in CLASSIFICATION_METRICS.values():
         for y_true, y_score in invalids:
             assert_raise_message(ValueError,
-                                 "Can't handle mix of binary and continuous",
```

Comment:
This seems like an odd change. Is the old message not tested any more? Can you change the tests such that both the old message and the new message are tested?